evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -409,8 +409,9 @@ fn parse_json(
     // values is an array of items. Depending on the JsonPath key, they
     // can be many or a single item. An item can be a single value or
     // an entire JSON object.
+    dbg!(&values);
     if let Some(coercion_type) = coerce {
-        if values.len() != 1 || values[0].is_object() {
+        if !values.iter().filter(|value| value.is_object()).collect::<Vec<&&Value>>().is_empty() {
```

Comment:
you can use `any` here 
https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.any